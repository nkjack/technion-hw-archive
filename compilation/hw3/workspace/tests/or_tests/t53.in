void foo(int c){
return;
}

void main(){
foo(256b);
}
