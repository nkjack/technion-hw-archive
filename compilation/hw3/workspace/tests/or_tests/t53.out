---end scope---
foo - 0 preconditions
c INT -1
line 6: byte value 256 out of range
